docs/learning/useful-resources/setup-new-github-repository.md

```diff
@@ -0,0 +1,59 @@
+# Setup new GitHub repository
+
+Everyone at Thoth Tech has permission to create new GitHub repositories. To ensure that we adhere to
+the
+[Git Contribution guide](https://github.com/thoth-tech/handbook/blob/main/docs/processes/quality-assurance/git-contribution-guide.md),
+we suggest to implement the following GitHub settings.
+
+## Deleted hard branches automatically
```

Comment:
Typo - "**Delete** hard branches automatically" or alternatively "Hard branches deleted automatically"?

Reply:
Oops it is supposed to be `Delete merged branches automatically` 😅

Reply:
Addressed in https://github.com/thoth-tech/handbook/pull/94/commits/2d656af3ec9e1d2635945621d3f94342a4385a20.